### PR TITLE
Change alias 'three-nodes' → 'three/nodes'

### PR DESCRIPTION
**Pull request by donmccurdy**
Related: 

- https://github.com/mrdoob/three.js/issues/24377
- https://github.com/mrdoob/three.js/pull/23368

In my opinion, any aliases in exported modules should be under the `three/*` namespace. Otherwise, it looks like an import from an unknown NPM module to tooling. If we'd like to do so, we could add the `three/nodes` entry in `package.json#exports`, allowing bundlers and some CDNs to resolve this import automatically, but I haven't taken that extra step here.

/cc @sunag @marcofugaro @robertlong

### CODE REVIEW

**Review by sunag — APPROVED** [review]
I think it's a positive change due to the many reports of issues.

File: examples/jsm/node-editor/utils/PreviewEditor.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import { OrbitControls } from 'three-addons/controls/OrbitControls.js';
-import { ViewHelper } from 'three-addons/helpers/ViewHelper.js';
+import { OrbitControls } from '../../controls/OrbitControls.js';
```
Comment (LeviPesin) [inline examples/jsm/node-editor/utils/PreviewEditor.js:1]: Why do you change here `three-addons` to `../..`?

File: examples/jsm/renderers/webgpu/WebGPUBackground.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { GPULoadOp, GPUStoreOp } from './constants.js';
 import { Color, Mesh, BoxGeometry, BackSide } from 'three';
-import { context, transformDirection, positionWorld, modelWorldMatrix } from 'three-nodes/Nodes.js';
-import MeshBasicNodeMaterial from 'three-nodes/materials/MeshBasicNodeMaterial.js';
+import { context, transformDirection, positionWorld, modelWorldMatrix } from 'three/nodes';
+import { MeshBasicNodeMaterial } from 'three/nodes';
```
Comment (LeviPesin) [inline examples/jsm/renderers/webgpu/WebGPUBackground.js:4]: I think these two imports can be merged into one.

**Comment by LeviPesin** [comment]
Also related: https://github.com/mrdoob/three.js/pull/23406

File: examples/jsm/renderers/webgpu/WebGPUBackground.js (hunk 2 above)
Comment (donmccurdy) [reply examples/jsm/renderers/webgpu/WebGPUBackground.js]: Thanks, good catch!

**Comment by marcofugaro** [comment]
Makes sense, thanks for the PR! 👍

File: examples/jsm/node-editor/utils/PreviewEditor.js (hunk 1 above)
Comment (LeviPesin) [reply examples/jsm/node-editor/utils/PreviewEditor.js:1]: So we should add https://github.com/mrdoob/three.js/pull/23406 change of such `../..` in place of `three-addons` to `three/addons`.

Comment (donmccurdy) [reply examples/jsm/node-editor/utils/PreviewEditor.js:1]: Perhaps. #23406 is a bigger change to the public API, and it sounds like people are still considering multiple npm packages, so I don't know what will happen there. But I don't think these couple of `three-addons` references are serving a necessary purpose in the meantime.

Comment (marcofugaro) [reply examples/jsm/node-editor/utils/PreviewEditor.js:1]: Yeah let's keep these two PRs separate.

Comment (LeviPesin) [reply examples/jsm/node-editor/utils/PreviewEditor.js:1]: No, I'm not suggesting to combine these PRs, I just think that if we are making such change in this PR we should also add a corresponding change to that one.

Comment (donmccurdy) [reply examples/jsm/node-editor/utils/PreviewEditor.js:1]: Let's discuss there — not a big deal if https://github.com/mrdoob/three.js/pull/23406 misses one or two references to a `../../` prefix that could have been replaced with the alias though. 

Goal of this current PR is to eliminate use of incompatible aliases.

**Comment by marcofugaro** [comment]
@donmccurdy could you solve the conflicts on this PR?

Also I think this PR should be merged soon to avoid any more confusion for the users.

**Comment by sunag** [comment]
Updated!

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**